src/bin.js

```diff
@@ -22,6 +22,7 @@ program
   .option("-b, --browsers <items>", "browsers list (comma separated)")
   .option("-I, --no-import", "do not inline @import")
   .option("-U, --no-url", "do not adjust url()")
+  .option("-e, --extract", "output values of custom properties")
```

Comment:
Oh I mean "into a file" :)
So lets go for "extract resolved custom properties and custom media into a file".
"medias" is a typo, this word does not exist in english. Media is already plural. This need to be fixed :)